mm_cataloguer/index_netcdf.py

```diff
@@ -506,7 +506,7 @@ def find_or_insert_level_set(sesh, cf, var_name):  # get.level.set.id
 
 def wkt(proj4):
     """Return a WKT representation of a CRS defined in PROJ.4 syntax."""
-    return pycrs.parser.from_proj4(proj4).to_ogc_wkt()
+    return pycrs.parse.from_proj4(proj4).to_ogc_wkt()
```

Comment:
Wut? Did we get a different version of the package that changed the name of the module?

Reply:
Apparently so. At least it wasn't working for me until I made that fix.